akka-docs/src/main/paradox/additional/rolling-updates.md

```diff
@@ -1,5 +1,12 @@
 # Rolling Updates
 
+@@@ note
+
+There are a few instances [when a full cluster restart is required](deploying.md#when-shutdownstartup-is-required)
```

Comment:
This seems like a dead link (but isn't caught by paradox because it doesn't have `@ref`)